lib/models/search-result.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+/**
+ * @module models/search-result
+ */
+
+var EventContext = require("./event-context");
+var utils = require("../utils");
+
+/**
+ * Construct a new SearchResult
+ *
+ * @param {number} rank   where this SearchResult ranks in the results
+ * @param {event-context.EventContext} event_context  the matching event and its
+ *    context
+ *
+ * @constructor
+ */
+function SearchResult(rank, event_context) {
```

Comment:
Nit: Prefer `camelCase` where possible. The only places where we should be using `under_scores` is when we're directly manipulating JSON responses from the server which are, of course, with underscores.